vector/src/main/res/values/strings.xml

```diff
@@ -2860,6 +2860,14 @@
     <string name="error_voice_message_cannot_reply_or_edit">Cannot reply or edit while voice message is active</string>
     <string name="voice_message_reply_content">Voice Message (%1$s)</string>
 
+    <string name="a11y_audio_message_item">%1$s, %2$s, %3$s</string> <!-- filename, duration, file size -->
```

Comment:
Comments have to be above the string for Weblate to handle it properly.

Reply:
Added this to https://github.com/vector-im/element-android/issues/5714